src/style.css

```
@@ -0,0 +1,54 @@
+body {
+  margin: 0%;
+ 
+}
+
+.container {
+  width: 96px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  
+}
+
+.stars {
+  
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  height: 16px;
+  
+}
```

```suggestion
.stars {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
  height: 16px;
}
```
Не оставляй пустые линии